### PR TITLE
HelloWorld on the sample test-network

### LINKED ISSUE #605: Improve test-network sample to accept any chaincode
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

The current test-network sample only accepts the echo chaincode. It could explain how to also use the helloworld chaincode or any other chaincode. To be able to do so the user would have to be able to configure the chaincode name and the path to the code itself.
**Describe alternatives you've considered**

The solution could be done in 2 steps. 
1. Standardize the use of CC_ID because in some parts TEST_CC_ID is used, add CC_PATH as configurable, update the Readme to reflect this change. 
2. Update just the readmes of the test-network and the helloworld samples to reflect the steps needed to use the helloworld chaincode in the test-network.
**Additional context**

### PR DESCRIPTION
**Pull request by munapower**
**What this PR does / why we need it**:
This improves the test-network sample to show how to change from echo chain code to helloworld chain code.
**Which issue(s) this PR fixes**:

Fixes #605 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing changes and/or breaks backward compatability?**:

```
This changes the Test-network sample Readme adding a section to explain how to change the chain code and test it afterwards. It also fixes how the path to the chain code was defined.

### CODE REVIEW

File: samples/deployment/test-network/README.md [hunk 1]
```diff
@@ -189,6 +189,19 @@ cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
 ./network.sh down
 ```
 
+## Changing chaincode used by test-network sample
```
Comment (mbrandenburger) [inline samples/deployment/test-network/README.md:192]:
```suggestion
## FPC HelloWorld on the test-network
```

File: samples/deployment/test-network/README.md [hunk 2]
```diff
@@ -189,6 +189,19 @@ cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
...
+If you have already reached this point you can try changing the chaincode used by FPC. For example, you can use the HelloWorld code instead of the echo code. To do so you have to change the values of the environment variables set at the beginning `CC_ID` and `CC_PATH`.
```
Comment (mbrandenburger) [inline samples/deployment/test-network/README.md:193]: Here we should link to the other tutorial. Please use a relative markdown link.

Comment (mbrandenburger) [reply samples/deployment/test-network/README.md:193]: What about also adding a reference to this section in the hello world tutorial?

Comment (mbrandenburger) [reply samples/deployment/test-network/README.md:193]: > If you have already reached this point you can try changing the chaincode used by FPC.

Maybe .... 
"In this section we show how you can run a different FPC chaincode on the test network. For example, ..."

**Review by mbrandenburger — CHANGES_REQUESTED** [review]
Thanks Maria! Thanks for this PR.
Please rebase on latest main and fix the DCO issue. :) Some suggestions for improvements.

File: samples/deployment/test-network/README.md (hunk 1 above)
Comment (munapower) [reply samples/deployment/test-network/README.md]: Changed title but not to suggested ... please review @mbrandenburger

File: samples/deployment/test-network/README.md (hunk 2 above)
Comment (munapower) [reply samples/deployment/test-network/README.md]: Changed text.

File: samples/deployment/test-network/README.md [hunk 3]
```diff
@@ -219,7 +232,10 @@ peer chaincode query -C mychannel -n ercc -c '{"Function": "queryListEnclaveCred
 E_ID=$(peer chaincode query -C mychannel -n ercc -c '{"Function": "queryListProvisionedEnclaves", "Args" : ["echo"]}' 2> /dev/null  | jq -r '.[0]')
 peer chaincode query -C mychannel -n ercc -c '{"Function": "queryEnclaveCredentials", "Args" : ["echo", "'${E_ID}'"]}'
 ```
+<<<<<<< HEAD
 
+=======
+>>>>>>> db8976c... Update test-network sample to enable chaincode change
```
Comment (mbrandenburger) [inline samples/deployment/test-network/README.md:238]: that should not be here :)

**Review by mbrandenburger — CHANGES_REQUESTED** [review]

File: samples/deployment/test-network/README.md (hunk 3 above)
Comment (munapower) [reply samples/deployment/test-network/README.md:238]: just saw it, will fix

**Review by mbrandenburger — COMMENTED** [review]
LGTM! Still need to test it tmrw.

**Review by bvavala — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**